trivia.py

```diff
@@ -461,17 +484,24 @@ def _get_new_question(self):
         '''
         damaged_question = True
```

Comment:
Ohey, we should be using generators to fetch questions, not doing it this way.

I kinda want to rearchitect portions of triviabot.